plugins/woocommerce/tests/api-core-tests/README.md

```diff
@@ -127,25 +129,279 @@ After you run a test, it's best to restart the environment to start from a fresh
+
+- Basic positive tests (happy paths) - check basic functionality and the acceptance criteria of the API
+- Extended positive testing with optional parameters - more thorough testing (can be used for testing a bug/updates/new functionality)
+- Negative testing with valid input - expect to gracefully handle problem scenarios with valid user input e.g. trying to add an existing username
+- Negative testing with invalid input - expect to gracefully handle problem scenarios with invalid user input e.g. trying to add a username which is null
+
+
 ### Creating test structure
 
-The structure of the test serves as a skeleton for the test itself. You can turn it into a test by using `describe()` and `it()` methods of Playwright:
+The structure of the test serves as a skeleton for the test itself. 
 
-- [`test.describe()`](https://playwright.dev/docs/api/class-test#test-describe) - creates a block that groups together several related tests;
+Each test file requires the `@playwright/test` module to be imported as follows:
+```js
+const { test, expect } = require( '@playwright/test' );
+```
+You can create a test by using the `test.describe()` and `test()` methods of Playwright:
+
+- [`test.describe()`](https://playwright.dev/docs/api/class-test#test-describe-1) - creates a block that groups together several related tests;
 - [`test()`](https://playwright.dev/docs/api/class-test#test-call) - actual method that runs the test.
 
 Based on our example, the test skeleton would look as follows:
 
 ```js
-test.describe( 'Merchant can create virtual product', () => {
-	test( 'merchant can log in', async () => { } );
+test.describe( 'Coupons API tests', () => {
+	test( 'can create a coupon', async ( {request} ) => {
+    // test to create a coupon here
+  } );
+
+	test( 'can retrieve a coupon', async ( {request} ) => {
+    // test to retrieve a coupon here
+  } );
+
+	test( 'can update a coupon', async ( { request } )  => {
+    // test to update a coupon here
+  } );
+} );
+```
 
-	test( 'merchant can create virtual product', async () => { } );
+Note: you can also nest a `test.describe()` inside a `test.describe()`. Example:
 
-	test( 'merchant can verify that virtual product was created', async () => { } );
-} );
+```js
+test.describe('Orders API tests: CRUD', () => {
+	let orderId; //test variable
+
+	test.describe('Create an order', () => {
+		test('can create a pending order by default', async ({request}) => {
+      //test code here
+    }
+```
+This allows you to further subgroup tests. When viewing the tests results locally, each test describe 'level' will be separated by `>` in the console as below: 
+
+`Orders API tests: CRUD › Create an order › can create a pending order by default`
+
+## Test Data Setup/Teardown
+
+You may need test data setup prior to the execution of your tests and removed after the execution of your tests. This can be achieved with any of the following methods, depending on the needs of the test:
```

Comment:
I think "You may need test data setup prior to the execution of your tests and `make sure it is removed` after the execution of your tests" is more readable for me as a non-native English speaker.